tests/client/api_events/test_milestones_confirmed_event.c

```diff
@@ -0,0 +1,89 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#include <string.h>
+#include <unity/unity.h>
+
+#include "client/api/events/node_event.h"
+#include "client/api/events/sub_milestones_confirmed.h"
+#include "events_test_config.h"
+
+void setUp(void) {}
+
+void tearDown(void) {}
+
+void test_milestones_confirmed_parser(void) {
+  char *json_data = "{\"index\":1139767,\"timestamp\": 1637221249}";
+
+  // Test for expected events response
+  milestone_confirmed_t *res = res_milestones_confirmed_new();
```

Comment:
don't think dynamic allocation is needed for this response object. 
the user can simply use it like
```
milestone_confirmed_t res = {};
parse_milestones_confirmed(json_data, &res)
```
so `res_milestones_confirmed_new` and `res_milestones_confirmed_free` are useless.

Reply:
Removed dynamic allocation for milestones_confirmed and milestones_lates topic.